app/utils/config.py

```diff
@@ -4,8 +4,13 @@
 
 
 class Settings(BaseSettings):
+    postgres_user: str
+    postgres_password: str
+    postgres_db: str
+    database_url_sync: str
+    database_url_async: str
 
-    stac_url: str = "http://localhost:8080"
+    stac_url: str = "http://172.191.168.255:8082"
```

Comment:
The value of this environment variable must be set in the .env file, I consider that in the config file it should be kept as it was before